README.md

```diff
@@ -1 +1,132 @@
-# Final-capstone-project
\ No newline at end of file
+# Final-capstone-project
+
+# Power Spokes
+
+Power-Spokes is a booking app for motorcycles where users get to book a reservation for a motorcycles
+
+You can see the backend for this project(https://github.com/Addisu87/capstone-appointment-back-end).
+
+## Kanban Board
+
+[Kanban Board Link]()
+
+- We are four (3) in this team.
+
+  - @Emmykage
+  - @Isaacpitwa
+  - @Addisu87
+
+### Our Kanban board
+
+https://github.com/users/Emmykage/projects/4/views/1
+
+## Built With
+
+- React
+- Redux
+- Deployed with Netlify
+
+## Live Demo & Screenshots
+
+> [POWER-STROKE](https://final-capstone-project-udqc.vercel.app/)
+
+<table>
+<tr>
+<td><img src="src/img/splash.png" alt="splash page"></td>
+<td><img src="src/img/signup.png" alt="signup page"></td>
+<td><img src="src/img/login.png" alt="login page"></td>
+</tr>
+</table>
+<div align="center"><img src="src/img/all-versions-picture.png" alt="all pages"></div>
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Any web browser (e.g. Google Chrome)
+- Any Code Editor (e.g. Vscode)
+- NodeJS
+
+### Setup
+
+- Create a new folder.
+
+```bash
+mkdir folder
+```
+
+- Move to the folder:
+
+```bash
+cd folder
+```
+
+- Clone the repo:
+
+```bash
+git clone https://github.com/Emmykage/Final-capstone-project.git
+```
+
+### Install
+
+- Install the dependencies:
+
+```
+npm install
+```
+
+### Usage
+
+- Start a live development server:
+
+```
+npm start
+```
+
+### Run tests
+
+- Run tests:
+
+```
+npm test
+```
+
+## Authors
+
+👤 **Addisu Haile**
+
+- GitHub: [@Addisu87](https://img.shields.io/badge/-Addisu87-white?logo=GitHub&logoColor=181717&style=plastic)
+- Twitter: [@AddisuTedla](https://img.shields.io/badge/-AddisuTedla-white?logo=Twitter&logoColor=1DA1F2&style=plastic)
+- LinkedIn: [addisu-tedla](https://img.shields.io/badge/-addisu_tedla-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)
+
+## 👤 Isaac Pitwa
+
+- GitHub: [@isaacpitwa](https://github.com/isaacpitwa)
+- Twitter: [@isaacpitwa](https://twitter.com/isaacpitwa)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/isaac-pitwa)
+
+## 👤 Menanya Morris
+
+- GitHub: [@emmykage](https://github.com/emmykage)
+- Twitter: [@emmykage](https://twitter.com/emmykage)
+- LinkedIn: [Menanya Morris](https://www.linkedin.com/in/morris-menanya-a51985104/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Emmykage/Final-capstone-project/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Original design idea by: [Murat Korkmaz](https://www.behance.net/muratk)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

 ## Tips and extra help

Go [here] (https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪